tslint.json

```diff
@@ -19,15 +15,13 @@
     "prefer-method-signature": true,
     "arrow-parens": false,
     "no-unnecessary-type-assertion": true,
-    "array-type": [
-      true,
-      "array"
-    ],
+    "array-type": [true, "array"],
     "interface-name": false,
     "no-duplicate-imports": true,
     "no-console": false,
     "no-var-requires": false,
     "jsx-wrap-multiline": false,
+    "jsx-no-lambda": false,
```

Comment:
We would prefer to restrict lambda's.

Reply:
I took the lambdas out but whats the reasoning behind the preference

Reply:
Stylistic, but can have some performance improvements eventually if you
start heavily optimizing.
https://stackoverflow.com/questions/43968779/are-lambda-in-jsx-attributes-an-anti-pattern/43968902#43968902